core/src/main/java/io/grpc/internal/PickFirstLoadBalancerProvider.java

```diff
@@ -50,6 +50,8 @@ public String getPolicyName() {
 
   @Override
   public LoadBalancer newLoadBalancer(LoadBalancer.Helper helper) {
+    //  health check pick first load balancer in multi child LB policy.
+    //  dependency issue: health util is in service package.
```

Comment:
I thought that the plan was only to provide health checking on the new PF.

Reply:
Yes. The comment seems in wrong class but is still right: the idea is that we provide health checking by default in all the petiole policies, i.e. in MultiChildLbPolicy. But not at top level PickFirst.